src/components/Game/Letters/Letters.js

```diff
@@ -9,9 +10,12 @@ const Letters = ({ letters, clicked, dragDisabled, dragEnd }) => (
     <DragDropContext onDragEnd={dragEnd}>
         <section className={classes.Letters}>
             <Droppable droppableId="droppable" direction="horizontal">
-                {(provided) => (
+                {(provided, snapshot) => (
                     <ul
-                        className={classes.LettersContainer}
+                        className={classNames(
+                            classes.LettersContainer,
+                            snapshot.isDraggingOver ? classes.DraggingOver : null,
```

Comment:
or probably `{ [classes.DraggingOver]: snapshot.isDraggingOver }`
but it's okay right now :)